apps/vr-tests-react-components/src/stories/Textarea.stories.tsx

```diff
@@ -1,9 +1,11 @@
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27104&commitId=20ea54f8c3d6ce671f36fb2f82270dcb87b01c75&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1868 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Textarea Converged.contentAbove.default.chromium.png|0| Added|
|Textarea Converged.contentBelow.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>